### ISSUE TITLE
NTR: Breast biopsy: adenoma  (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9142]
New term request Breast biopsy: adenoma  (elthat_100318030616):

**Comment by pnrobinson** [comment]
@MickeySegal
There are several kinds of adenoma but the most common one is already in the HPO:
[Fibroadenoma of the breast HP:0010619](https://hpo.jax.org/app/browse/term/HP:0010619)
Is this what is needed?

**Comment by MickeySegal** [comment]
That works for many of our entries, but not others: https://www.ncbi.nlm.nih.gov/books/NBK1286/ refers to "Breast ductal adenoma" 

https://surgpathcriteria.stanford.edu/breast/ductaladenomabr/printable.html has a list of various breast adenomas:
[Apocrine adenoma](https://surgpathcriteria.stanford.edu/breast/apocrineadenomabr/)
[Ductal adenoma](https://surgpathcriteria.stanford.edu/breast/ductaladenomabr/)
[Fibroadenoma](https://surgpathcriteria.stanford.edu/breast/fibroadenoma/)
[Infiltrating syringomatous adenoma](https://surgpathcriteria.stanford.edu/breast/syringadbr/)
[Juvenile fibroadenoma](https://surgpathcriteria.stanford.edu/breast/juvfibroadenoma/)
[Nipple adenoma](https://surgpathcriteria.stanford.edu/breast/nippleadenoma/)
[Lactating adenoma](https://surgpathcriteria.stanford.edu/breast/lactadbr/)
[Pleomorphic adenoma](https://surgpathcriteria.stanford.edu/breast/pleoadbr/)
[Tubular adenoma](https://surgpathcriteria.stanford.edu/breast/tubularadbr/)

**Comment by pnrobinson** [comment]
OK, for now I added this:

Adding new term: 
- ID: HP:6000894
- label: Breast adenoma
- parents: [Neoplasm of the breast (HP:0100013)](https://hpo.jax.org/app/browse/term/HP:0100013)
- pmid: 
- def: A benign, well circumscribed neoplasm that arises from the breast.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9142
- synonyms: 


We will be returning to the histology this year.
